Updated ALiAS FAQs (#151)
asetalias/asetalias.github.io: ALiAS_FAQs.md

                                                FAQs
Q1. What is ASET AliAS?

> Ans. It’s not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the Open Source. We community members help students with their doubts pertaining tech, conduct workshops, webinars, have meet-ups (online and in-person) and most important get you connected.

Reviewer: ```suggestion
Ans. **ALiAS** is not your regular Amity Club. ALiAS is a community, founded in 2010, which was developed with the aim to promote the use of Linux in the campus and get students familiar with the world of Open Source. The community members help students with their doubts pertaining to tech, conduct workshops, webinars, organize meet-ups (online and in-person), and most important get you connected.
```

Reviewer: the open source --> either add the world of or remove the `the`
most important get  you connected --> I don't have a suggestion rn but that just doesn't feel right

Q2. Why should I join ALiAS?

> Ans. ALiAS is by far, the largest open-source community in Amity University, we are

Reviewer: ```suggestion
Ans. ALiAS is by far, the largest open-source community in Amity University. We are
```

open to developers not only inside the campus, but also from various places. We have got assistance

> from alumni and people from tech, near and far. We have got presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.

Reviewer: ```suggestion
from alumni and people from tech, near and far. We have got a presence in communities like PyDelhi, ILUG-D, LinuxChix India, Mozilla Delhi, etc. and can help you get connected.
```
`people from tech, near and far` 
We can do better than that

`We have got presence in communities`--> Many of our members are also members of communities
That still doesn't sound that good xD
That can be done better

Q3. How is the community of AliAS?

> And. We have got the most active developers in the campus. You will find people in nearly all domains,

Reviewer: ```suggestion
Ans. We have the most active developers in the campus. You will find people from nearly all domains,
```
It should be either `we have the most active developers` or `we got the maximum number of active developers of any group/club` Something like that

> Web Development, Android Development, Artificial Intelligence, backend, frontend, industry people,

Reviewer: ah... industry people !??

We can also add Designers, Technical Writers, theoretical CS  enthusiasts or something 
instead of just focusing on developers

> etc. And best of all, we have got a great reach for people and further development.

Reviewer: That looks like an improper sentence.`and further development` doesn't fir that well

Q4. How should I join AliAS?

> And. Simple. Go and register yourself in http://asetalias.in/ . One of our developers will get you joined

Reviewer 2: Any one of us will add you to the group.

Reviewer 3: Define Us, because I haven't seen any new members being added.

Reviewer: Then you can assign one person for that job

Reviewer 3: Last I remember, Siddharth was working on this.

> And. Simple. Go and register yourself in http://asetalias.in/ . One of our developers will get you joined

Reviewer: ```suggestion
Ans. Simple. Go and register yourself on http://asetalias.in/ . One of us will add you to the WhatsApp and Telegram groups.
```

to the group.

Q5. Having joined, where should I start?

> And. We request you to drop a message, stating your name, course duration and your skills, if any. Also

Reviewer: ```suggestion
Ans. We request you to drop a message, stating your name, course duration, and your skills. Also
```
Bro, `if any` is rude lol

Author: Rude? Ever saw any freshers batch in Amity? Nothing can be more disappointing than that. People can't even understand simple print statements! Although, it had a satirical remark, they literally don't know anything. But okay. Accepted XD

Reviewer: I never said it was wrong, just rude. If someone is bad at something, doesn't mean you have to confront them about it :rofl:  especially when you're introducing them to a new community.

Reviewer: Also, everyone starts like that

Reviewer 3: Being humble every time, everywhere is everything in an open-source community. No one is born with the knowledge of programming, no one understands at the same rate. Hence, we cannot, should not and will not pass judgment on the basis of skills, experience, age, and especially if there are freshers.

> Ever saw any freshers batch in Amity? Nothing can be more disappointing than that. People can't even understand simple print statements!

Well, that means I would be pretty disappointing then. Before joining Amity, I was the technical head of my school's technical society even when I didn't opt for Computer Science as my 5th. I won many awards for website design. Never wrote a single line of code. I had a hard time getting over the barrier to understand basic shit. This wouldn't have been possible if Shyam bhaiya would have been disappointed in me before giving me a single chance. Give the people a chance that they deserve, remain humble, be positive. Carry on...

Author: Okay... :pray:

> let us know which field you want to work, and we will guide you thereafter. Have one plan at a time.

Reviewer: ```suggestion
let us know which field you are interested in, and we will guide you thereafter. Have one plan at a time.
```
`Have one plan at a time` doesn't flow well with the rest of the message

> Q6. Best OS?

Reviewer: ```suggestion
Q6. What Operating System should I use? How will it affect my work?
```
or something like that

Reviewer 3: Quite better.

> Ans. Are you a designer? - Windows; Are you a developer? - Linux(Any)

Reviewer: ```suggestion
Ans. Are you a designer? - Windows; Are you a developer? - Linux(Any)
```
Woah, not that simple bro lol

Author: This answer left to @vipulgupta2048

Reviewer 3: Yes, and I leave this answer to anyone who would like to take a shot at it. 
Quite frankly, this doesn't have straight answer.
@Sc0rpi0n101 You can give it a shot. Collaborate in this PR with @sachanakshat and add your contribution too.

Reviewer: Sure

Windows has great graphics. It spoon feeds you everything. Many design softwares, games, drivers and
support is easily available in windows. Installing anything is a cup of cake for anyone. But if you want
to delve into softwares deeper, please forget your games and ease. Being open source community, we
recommend installing Linux as dual boot, and get into ease with it. The only difference you will see in
different distros is the GUI. Some software packages and libraries are pre-installed in some distros,
while some requires you to install. Install Ubuntu or Linux Mint if you are a beginner. Ubuntu has got
best support for you. (Pro-tip: Give more space to Home directory as all your files are installed here).

Q7. The language I should start with?

> Ans. Start with C++. Next learn JAVA, then Python. This will fulfill most of your software needs and

Reviewer: ```suggestion
Ans. Start with C++. Next learn JAVA, then Python. This will fulfill most of your software needs and
```
Broo, C++ is **NOT** a good first language.
It's too complex for beginners.
Start with a general purpose, easy to learn, currently in fashion language.
Then maybe suggest 2nd languages based on their domain of choice

Author: Will C work?

Reviewer: Ahh...NO, C is worse than C++ 
Try something easy like python/ruby/javascript, some scripting language
C++ and Java are good for 2nd language when people have some idea what they're doing
C is the real s***. Ir's for later :rofl:

problems.

P.S. - Please DON’T ASK TO ASK, JUST ASK. We all are learners here, and all of us are noobs in
some field or the other. Don’t be shy. If you don’t know, just admit it, and ask and learn to be better
you.

Join us-

> email: contactasetalias@gmail.com

Reviewer 2: email -> Email
